extension/memorylimiterextension/config.go

```diff
@@ -0,0 +1,42 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package memorylimiterextension // import "go.opentelemetry.io/collector/extension/memorylimiterextension"
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/internal/memorylimiter"
+)
+
+// MemoryLimiterExtension is an Extension control memory usage of OpenTelemetry Collector.
+// Common usage is to check memory usage during connection and can reject or accept the connection based on the
+// configured memoryLimiter settings
+type MemoryLimiterExtension interface {
```

Comment:
Let's leave this out for now. This doesn't belong to `config.go`. The extension can be checked as `ext.(interface{ MustRefuse() bool })` anyway.

Reply:
removed